qa/suites/rados/singleton/all/ec-lost-unfound-upgrade.yaml

```diff
@@ -26,5 +26,7 @@ tasks:
 - print: "upgraded mon.a and friends"
 - ceph.restart:
     daemons: [mon.a, mon.b, mon.c, osd.0, osd.1, osd.2]
+- sleep:
+    duration: 300 # http://tracker.ceph.com/issues/16239
```

Comment:
@smithfarm maybe 20 would suffice. i think the reason why the osd is not ready for "tell" command after "ceph health" shows that the cluster is "HEALTH_OK" is that the monitor fails to be notified that the osd in question is not up in "heatbeat_interval". because infernalis does not have the `osd_fast_fail_on_connection_refused` support, the monitor need longer to detect an osd down.  and `osd_heartbeat_grace` is used to determine if an osd is down.